sections/community_brokers.tex

```diff
@@ -189,3 +199,6 @@ \subsection{Timeline}
 		\end{itemize}
 	\item \textbf{Full LSST operations} \textit{currently Q4 2022}
 \end{itemize}
+}
+
+\newtext{\textbf{updated timeline with estimates of DPs, commissioning,  LOY1 ...}}
```

Comment:
We should update this just before issue with the latest ComCam / LSSTCam /SV  survey schedule. The current Data Preview schedule can be found at https://docs.google.com/presentation/d/1tRHdSyXBRp850zfJO5NssHS-fGfGMMngDEfAD9Azdk4/edit#slide=id.p1